### PR TITLE
[RW-4825][risk=no] Enhance metadata on workspace for egress alert

### PR DESCRIPTION
**Pull request by jaycarlton**
Putting some more metadata on the users and workspaces in the workspaces admin endpoint. This gives information about 
* the cluster
* the user running it
* the workspace's age
* collaborators

There's a focus on human-centric details for the first responder to use, since most of the responses to the egress alerts going forward will benefit from information about the actual users and institutions.

Some refactoring to make this data more available.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                    "Collaborator with user_id %d not Found", userAdminView.getUserDatabaseId()));
+    return String.format("%s: %s", userAdminView.getRole(), userDetails);
+  }
+
+  /**
+   * Produce a succinct string for giving an administrator a heads-up in an alert or other text-only
+   * context.
+   */
+  @Transient
+  public String getAdminDescription(DbUser dbUser) {
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java:176]: @calbach per our discussion, I've removed the contact email. I believe the remainder of the fields are in the researcher directory (name & institution).

I wonder if we need a global identifier for users that's completely anonymous for some purposes.

Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java:176]: Thanks.

File: api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java [hunk 2]
```diff
@@ -106,4 +108,13 @@ boolean validateAffiliation(
...
    */
   boolean validateOperationalUser(DbInstitution institution);
+
+  /**
+   * @deprecated
+   * @param institutionalAffiliation
+   * @return
+   */
+  @Deprecated
+  DbInstitutionalAffiliation legacyInstitutionToDbInstitution(
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java:118]: `legacyAffiliationToDbAffiliation`

**Comment by jaycarlton** [comment]
Discussed with @jmthibault79. Decision is to:
* have a PR with just prerequisites for the ticket at hand
* second PR with implementation of the functionality
* possible third PR with Profile refactorings, but need to work with Joel b/c of plans for & work in this area.

**Pull request closed without merging**